security.sh

```diff
@@ -1,13 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Is the security.sh needed for the jenkins security checks?

Reply:
when security.sh isn't found, it pulls it from the HMCTS repo directly so uses the most up to date one